src/components/card/category/categoryCard.jsx

```diff
@@ -1,12 +1,15 @@
+import { useGlobal } from "context/globalContext"
 import { Link } from "react-router-dom"
 
 export const CategoryCard = ({category}) => {
-    const {type,img} = category
+    const {type,img,id} = category
+    const {globalDispatch} = useGlobal()
+    
```

Comment:
Youe can also destructure like this : 
 export const CategoryCard = ({category:{type,img,id}) =>{}